src/main/java/com/google/sps/servlets/SpoonacularCombinedServlet.java

```diff
@@ -38,7 +38,7 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String query = request.getParameter("dishName");
     Client client = ClientBuilder.newClient();
-    WebTarget target = client.target(spoonacularPrefix + "/search?query=" + query + "&number=4&includeNutrition=true&apiKey=" + spoonacularAPIKey);
+    WebTarget target = client.target(spoonacularPrefix + "/search?query=" + query + "&number=6&includeNutrition=true&apiKey=" + spoonacularAPIKey);
```

Comment:
nit: Considering making the number value a constant, also consider using String.format()